### DISCUSSION TITLE
Execute CTE Query

### DISCUSSION THREAD

**Discussion by MadamArk** [doctrine/doctrine2#9279] [Support Questions] [answered]
Hello, I would like to know if it is possible to execute CTE requests like this one via Doctrine or Symfony :
"WITH RECURSIVE ancestors AS
                   (SELECT * FROM place
                     WHERE id=1
                     UNION
                     SELECT f.*
                     FROM place AS f, ancestors AS a
                     WHERE (f.id_parent_id = a.id OR f.id = a.id_parent_id))
SELECT * FROM ancestors order by id asc"

Thank you in advance for your answers

Translated with www.DeepL.com/Translator (free version)

**Comment by beberlei** [comment, marked as answer]
Its not possible to do CTE queres with DQL, but you can use SQL and result set mapping:

https://www.doctrine-project.org/projects/doctrine-orm/en/2.10/reference/native-sql.html#native-sql
